powerauth-load-tests/src/test/java/com/wultra/security/powerauth/test/config/PowerAuthLoadTestCommon.java

```diff
@@ -113,6 +132,44 @@ private static int getIntEnv(final String name, final int defaultValue) {
         return defaultValue;
     }
 
+    /**
+     * Fetches a boolean environment variable by name, or returns the default value if not found or invalid.
+     *
+     * @param name         The name of the environment variable.
+     * @param defaultValue The default value to use if the environment variable is not found or invalid.
+     * @return The environment variable value as a boolean, or the default value.
+     */
+    private static boolean getBooleanEnv(final String name, final boolean defaultValue) {
+        final String value = System.getenv(name);
+        if (StringUtils.isNotBlank(value)) {
+            if (value.equalsIgnoreCase("true") || value.equalsIgnoreCase("false")) {
+                return Boolean.parseBoolean(value);
+            } else {
+                logger.warn("Environment variable {} is not a valid boolean: {}. Using default value.", name, defaultValue);
```

Comment:
👀 Default value is logged, but the message sounds to me, like you wanted to print the env value. So what was your intention?